### ISSUE TITLE
ASR Transcription Bug

### ISSUE THREAD

**Issue by bruceye777** [murtaza-nasir/speakr#100]
Why is it that after I enabled ASR transcription, all Chinese audio is recognized as English regardless of whether I set the language to zh or leave it blank (auto detection)? 
Previously, when I used the OpenAI Whisper interface, it could be recognized normally.

<img width="639" height="577" alt="Image" src="https://github.com/user-attachments/assets/d620e5b5-24e2-4a49-a273-48e5e0538737" />

speakr container log
2025-09-20 15:27:15,301 - src.app - INFO - Sending ASR request to http://whisper-asr-webservice:9000/asr with params: {'encode': True, 'task': 'transcribe', 'output': 'json', 'language': 'zh', 'diarize': True} (timeout: 1800s)

asr container log
/app/.venv/lib/python3.10/site-packages/transformers/configuration_utils.py:309: UserWarning: Passing `gradient_checkpointing` to a config initialization is deprecated and will be removed in v5 Transformers. Using `model.gradient_checkpointing_enable()` instead, or if you are using the `Trainer` API, pass `gradient_checkpointing=True` in your `TrainingArguments`.
  warnings.warn(
/app/.venv/lib/python3.10/site-packages/pyannote/audio/models/blocks/pooling.py:104: UserWarning: std(): degrees of freedom is <= 0. Correction should be strictly less than the reduction factor (input numel divided by output numel). (Triggered internally at /pytorch/aten/src/ATen/native/ReduceOps.cpp:1839.)
  std = sequences.std(dim=-1, correction=1)
No language specified, language will be first be detected for each audio file (increases inference time).
>>Performing voice activity detection using Pyannote...
Model was trained with pyannote.audio 0.0.1, yours is 3.3.2. Bad things might happen unless you revert pyannote.audio to 0.x.
Model was trained with torch 1.10.0+cu102, yours is 2.7.1+cpu. Bad things might happen unless you revert torch to 1.x.
INFO:     172.21.0.2:38022 - "POST /asr?encode=true&task=transcribe&output=json&language=zh&diarize=true HTTP/1.1" 200 OK

docker-compose.yml
services:
  app:
    image: learnedmachine/speakr:latest
    container_name: speakr
    restart: unless-stopped
    ports:
      - "0.0.0.0:30009:8899"
    env_file:
      - .env
    environment:
      - LOG_LEVEL=INFO
    volumes:
      - ./uploads:/data/uploads
      - ./instance:/data/instance
    networks:
      - speakr-network

  whisper-asr:
    image: onerahmet/openai-whisper-asr-webservice:latest
    container_name: whisper-asr-webservice
    ports:
      - "9000:9000"
    environment:
      - ASR_MODEL=distil-large-v3
      - ASR_COMPUTE_TYPE=int8
      - ASR_ENGINE=whisperx
      - HF_TOKEN=xxxx
    restart: unless-stopped
    networks:
      - speakr-network

networks:
  speakr-network:
    external: true

**Comment by murtaza-nasir** [comment]
You need to use the large-v3 model, distil-large-v3 will not properly output Chinese.
